Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM simonjupp/ebi-ols:3.2.0
```

Comment:
Funny to see Simon personal repo as source :-)

Reply:
True, but that seems like the only source of that image on docker hub.